### PR TITLE
[LIBCLOUD-905] Added list images method for OnApp

### PR DESCRIPTION
**Pull request by hackedunit**
### Description
- Implements methods for listing available images for OnApp driver
- Includes tests and documentation

JIRA ticket: https://issues.apache.org/jira/browse/LIBCLOUD-905
Provider API doc: https://docs.onapp.com/display/52API/Get+List+of+All+Templates

### Status

- done, ready for review

### Checklist (tick everything that applies)

- [x] [Code linting](http://libcloud.readthedocs.org/en/latest/development.html#code-style-guide)
- [x] Documentation
- [x] [Tests](http://libcloud.readthedocs.org/en/latest/testing.html)
- [ ] [ICLA](http://libcloud.readthedocs.org/en/latest/development.html#contributing-bigger-changes)

### CODE REVIEW

File: libcloud/compute/drivers/onapp.py [hunk 1]
```diff
@@ -315,10 +315,37 @@ def list_nodes(self):
...
         return nodes
 
+    def list_images(self):
+        """
+        List all images
+
+        :rtype: ``list`` of :class:`NodeImage`
+        """
+        response = self.connection.request("/templates.json")
+        # return list(map(self._to_image, data['image_template']))
```
Comment (tonybaloney) [inline libcloud/compute/drivers/onapp.py]: please remove this commented line

**Review by tonybaloney — CHANGES_REQUESTED** [review]
changes LGTM, just 1 small comment left behind before it can be merged. thanks

**Comment by tonybaloney** [comment]
👍

**Comment by hackedunit** [comment]
@tonybaloney quick question - do I still need to attach a patch to the issue at JIRA as explained in the dev guide? Looks like these changes are already in trunk.

**Pull request closed without merging**